README.md

```diff
@@ -0,0 +1,14 @@
+# Contiv Prometheus Integration for Kubernetes
+
+## Getting Started
+
+In order to run this exporter for your cluster, please follow [this]() tutorial.
+
+## Exporter Architecture
+
+![Exporter Pipeline](exporter.png)
+
```

Comment:
For 4. Netplugin uses OpenvSwitch to program the traffic flows. Netplugin starts an ovsdb-server at port 6640. ovsdb stores information about the traffic flows and statistics.
Exporter can access the ovsdb at localhost:6640 as it is running in the same network namespace, as part of the same pod.